app/Image/SizeVariantDefaultFactory.php

```diff
@@ -345,20 +363,20 @@ protected function getMaxDimensions(int $sizeVariant): array
 	{
 		switch ($sizeVariant) {
 			case SizeVariant::MEDIUM2X:
-				$maxWidth = 2 * intval(Configs::get_value('medium_max_width'));
-				$maxHeight = 2 * intval(Configs::get_value('medium_max_height'));
+				$maxWidth = 2 * Configs::getValueAsInt('medium_max_width');
+				$maxHeight = 2 * Configs::getValueAsInt('medium_max_height');
```

Comment:
```suggestion
				$maxWidth = 2 * Configs::getValueAsInt('medium_max_width', 1920);
				$maxHeight = 2 * Configs::getValueAsInt('medium_max_height', 1080);
```

The more I add of these missing defaults, the more I wonder why we bother with them at all. Thoughts?